src/viadot/sources/base.py

```diff
@@ -290,7 +298,7 @@ def create_table(
         table: str,
         schema: str | None = None,
         dtypes: dict[str, Any] | None = None,
-        if_exists: Literal["fail", "replace", "skip", "delete"] = "fail",
+        if_exists: Literal["append", "replace", "skip", "fail"] = "fail",
```

Comment:
Hm but the logic didn't change, so these are incorrect. https://github.com/dyvenia/viadot/blob/cbf44560f4d867e4641ca5b28cc2202b07da0b99/src/viadot/sources/base.py#L319-L331

Reply:
Right, I added `delete` back, but added append